### PR TITLE
(documentation): updated application-state text

### PR DESCRIPTION
**Pull request by Paterma**
## Brief Description

Updated application state text on page as per instructions on design page stating "updated text in green". Reached out to Cole for clarification if that is all that needs to be changed, as his page is very different from the live page. Will update if necessary. 

## JIRA Link

https://rocketcom.atlassian.net/browse/ASTRO-4057

## Types of changes

- [ ] Bug fix
- [ ] New feature
- [ ] Breaking change

## Checklist

- [ ] This PR adds or removes a Storybook story.
- [ ] I have added tests to cover my changes.
- [ ] Regressions are passing and/or failures are documented
- [ ] Changes have been checked in evergreen browsers

### CODE REVIEW

File: packages/astro-uxds/_content/components/application-state.md [hunk 1]
```diff
@@ -35,15 +35,15 @@ The Application State component indicates the state the entire application is in
...
 ::: caution
 Only use Tag colors when necessary as overuse of colors may reduce efficacy of monitoring icon, classification marking, and notification banner component colors.
 :::
 
 ### Text
 
-- Text within the component uses the Medium body text size (14 px/0.875 rem) and sentence casing.
+- Text within the component uses the Body 2 style and sentence casing.
 
 - The default font color is the Dark Theme’s default body text color (#FFFFFF) for use with the default, dark background color.
```
Comment (micahjones13) [inline packages/astro-uxds/_content/components/application-state.md:49]: Pretty sure the `--colorTertiaryDarken4` will be dropped and instead be `color.palette.darkblue.950`. Third bullet point in the text section of https://rocketcom.atlassian.net/wiki/spaces/AS11/pages/2612789259/Application+State.

**Review by micahjones13 — CHANGES_REQUESTED** [review]
I did a quick comparison of all the text here to his confluence doc with the updates and only noticed the --colorTertiraryDarken4 swtich. Once that's in this looks good to me.

File: packages/astro-uxds/_content/components/application-state.md (hunk 1 above)
Comment (Paterma) [reply packages/astro-uxds/_content/components/application-state.md:47]: Yes! Just updated that, as well as all the other text on the page. Cole said anything (even not in green) needs to be changed. It should all be identical now

**Review by Paterma — COMMENTED** [review]
Updated necessary text

**Review by micahjones13 — APPROVED** [review]

**Pull request merged**